MiniApp/Classes/core/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -86,35 +86,52 @@ internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
+                        self.messageSentSuccessResponse(callBackId: callBackId, notEmptyId: notEmptyId)
+                    case .failure(let error):
+                        self.executeJavaScriptCallback(responseStatus: .onError, messageId: callBackId, response: error.localizedDescription)
                     }
-                    self.executeJavaScriptCallback(responseStatus: .onSuccess, messageId: callBackId, response: response)
-                case .failure(let error):
-                    self.executeJavaScriptCallback(responseStatus: .onError, messageId: callBackId, response: error.localizedDescription)
-                }
-            })
-        } else {
-            executeJavaScriptCallback(responseStatus: .onError, messageId: callBackId, response: getMiniAppErrorMessage(MiniAppJavaScriptError.unexpectedMessageFormat))
+                })
+            } else {
+                executeJavaScriptCallback(responseStatus: .onError, messageId: callBackId, response: getMiniAppErrorMessage(MiniAppJavaScriptError.unexpectedMessageFormat))
+            }
         }
     }
 
     private func sendMessageToContactId(with callBackId: String, parameters: RequestParameters?) {
-        if let message = parameters?.messageToContact {
-            if let contactId = parameters?.contactId {
-                hostAppMessageDelegate?.sendMessageToContactId(contactId, message: message) { result in
+        if isUserAllowedPermission(customPermissionType: .sendMessage, callbackId: callBackId) {
+            if let message = parameters?.messageToContact {
+                if let contactId = parameters?.contactId {
+                    hostAppMessageDelegate?.sendMessageToContactId(contactId, message: message) { result in
+                        switch result {
+                        case .success(let contact):
+                            let notEmptyId: String? = contact?.trimmingCharacters(in: .whitespacesAndNewlines).isEmpty ?? true ? nil : contact
+                            self.messageSentSuccessResponse(callBackId: callBackId, notEmptyId: notEmptyId)
+                        case .failure(let error):
+                            self.executeJavaScriptCallback(responseStatus: .onError, messageId: callBackId, response: error.localizedDescription)
+                        }
+                    }
+                } else {
+                    executeJavaScriptCallback(responseStatus: .onError, messageId: callBackId, response: getMiniAppErrorMessage(MiniAppJavaScriptError.unexpectedMessageFormat))
+                }
+            } else {
+                executeJavaScriptCallback(responseStatus: .onError, messageId: callBackId, response: getMiniAppErrorMessage(MiniAppJavaScriptError.unexpectedMessageFormat))
+            }
+        }
+    }
+
+    private func sendMessageToMultipleContacts(with callBackId: String, parameters: RequestParameters?) {
+        if isUserAllowedPermission(customPermissionType: .sendMessage, callbackId: callBackId) {
```

Comment:
Same here as above: we don't need to check the permission for `sendMessageToMultipleContacts`.